docs/application/web/tutorials/sec-privileges.md

```diff
@@ -19,214 +19,98 @@ Since Tizen platform 3.0, some privileges are categorized as privacy-related and
-
-**Table: Mobile Web Supplementary API privileges**
-
-| Privilege                               | Level  | Since | Description                              |
+<a name="API"></a>
+## Web API Privileges
+
+The following tables list the API privileges, which you must declare when using security-sensitive API modules in Web applications.
+
+**Table: Web Device API privileges**
+
+| Privilege                            | Level    | Privacy    | Description |
+|--------------------------------------|----------|------------|-------------|
+| `http://tizen.org/privilege/account.read` | public | Account | The application can read accounts. |
+| `http://tizen.org/privilege/account.write` | public | Account | The application can create, edit, and delete accounts. |
+| `http://tizen.org/privilege/alarm` | public |  | The application can manage alarms by retrieving saved alarms and waking the device up at scheduled times. |
+| `http://tizen.org/privilege/apphistory.read` | public | User history | The application can read the statistics of application usage, such as which applications have been used frequently or recently. |
+| `http://tizen.org/privilege/application.info` | public |  | The application can retrieve information related to other applications. |
+| `http://tizen.org/privilege/application.launch` | public |  | The application can open other applications using the application ID or application control. |
+| `http://tizen.org/privilege/appmanager.certificate` | partner |  | The application can retrieve specified application certificates. |
+| `http://tizen.org/privilege/appmanager.kill` | partner |  | The application can close other applications. |
+| `http://tizen.org/privilege/appmanager.launch` | public |  | The application can open other applications. |
+| `http://tizen.org/privilege/bluetooth` | public |  | The application can perform unrestricted actions using Bluetooth, such as scanning for and connecting to other devices. |
+| `http://tizen.org/privilege/bluetooth.admin` | public |  | The application can change Bluetooth settings, such as turning Bluetooth on or off, setting the device name, and enabling or disabling AV remote control. |
+| `http://tizen.org/privilege/bluetooth.gap` | public |  | The application can use the Bluetooth Generic Access Profile (GAP) to, for example, scan for and pair with devices. |
+| `http://tizen.org/privilege/bluetooth.health` | public |  | The application can use the Bluetooth Health Device Profile (HDP) to, for example, send health information. |
```

Comment:
The application can use the Bluetooth Health Device Profile (HDP). As an example it can send health information.

Reply:
done.